pandas/tests/util/test_show_versions.py

```diff
@@ -26,21 +33,44 @@
     "ignore:Distutils:UserWarning"
 )
 @pytest.mark.filterwarnings("ignore:Setuptools is replacing distutils:UserWarning")
-def test_show_versions(capsys):
+@pytest.mark.parametrize("as_json", [True, False, "test_output.json"], indirect=True)
```

Comment:
why do we need the indirect parameterisation of the fixture? There are no composite fixtures in play here?

probably better to parametrize the fixture directly, or just parameterise the test function and remove the fixture.

Reply:
yeah this is only used once, just paramterize directly